src/extension.ts

```diff
@@ -130,8 +131,15 @@ function displayLog(outputPanel: vscode.OutputChannel, message: string, show: bo
 }
 
 function executeCommand(command: (...args: any[]) => Promise<any>, thisArg: any, ...params: any[]) {
+    const errorMessage = params[params.length - 1];
     return command.call(thisArg, ...params).catch((err: string | Error) => {
         const error = typeof err === 'string' ? new Error(err) : err;
-        vscode.window.showErrorMessage(error.message);
+        let msg = error.message;
+        if (typeof err !== 'string') {
+            msg = `${errorMessage} Extension backend error - ${msg.toLowerCase()}`;
+        } else {
+            msg = `${errorMessage} ${msg.toLowerCase()}`;
```

Comment:
why is no 'Extension backend error' added if err is not a string?

Reply:
Most of the time string messages are generated by the extension itself while error objects from the server (e.g timeout) so it was a way to display the error nicely, but looking at the code better the label can work with 'Extension backend error' added in both cases.